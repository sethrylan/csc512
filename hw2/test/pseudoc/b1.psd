// source: http://en.wikipedia.org/wiki/Jasmin_(Java_assembler)#Example
BEGIN
WRITE(1138);
END
