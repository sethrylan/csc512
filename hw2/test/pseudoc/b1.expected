.source b1.psd 
.class public b1 
.super java/lang/Object 
; 
; standard initializer (calls java.lang.Object's initializer) 
; 
.method public <init>()V 
	aload_0 
	invokenonvirtual java/lang/Object/<init>()V 
	return 
.end method 
; 
; main() 
; 
.method public static main([Ljava/lang/String;)V 
	.limit stack 100 
	.limit locals 101 
	getstatic java/lang/System/out Ljava/io/PrintStream; 
	ldc 1138
	invokevirtual java/io/PrintStream/println(I)V 
	; done 
	return 
.end method 
